MacaoProject/Macao/Models/Card.cs

```diff
@@ -23,9 +23,15 @@ public Card(CardValueEnum value, CardSymbolEnum symbol, string image)
             Picture = image;
         }
 
-        public void Validation()
+        public bool Validation(Card card, Card firstCard)
         {
-
+            Player player = new Player();
```

Comment:
why you createa a player here? every time you validate a move you create a player? that does not make sense